package.json

```diff
@@ -6,10 +6,8 @@
 		]
 	},
 	"browserslist": [
-		"> 1%",
-		"last 2 versions",
-		"ie 11",
-		"not ie <= 8"
+		"defaults",
```

Comment:
is this the configuration for yoda as well?

Reply:
yup. this is some old set they have there - basically we need pollyfilling for ie11, and then for the common defaults, which cover all major browsers plus a bunch we don't use often but are quite common (opera etc.).